src/navsat_transform.cpp

```diff
@@ -179,14 +179,14 @@ void NavSatTransform::transformCallback()
       imu_sub_.reset();
     }
   } else {
-    std::unique_ptr<nav_msgs::msg::Odometry> gps_odom;
-    if (prepareGpsOdometry(gps_odom)) {
+    auto gps_odom = std::unique_ptr<nav_msgs::msg::Odometry>();
```

Comment:
If you're going to make it here, it should be make_unique not unique_ptr.

Reply:
Ya my bad. I missed this one when converting unique_ptr to make_unique.

Reply:
No worries, I read that a few times and kept thinking "what's wrong here... this looks right... oh yea".